src/peergos/server/net/DHTHandler.java

```diff
@@ -36,7 +36,7 @@ public DHTHandler(ContentAddressedStorage dht, BiFunction<PublicKeyHash, Integer
         this(dht, keyFilter, "/api/v0/");
     }
 
-    private Map<String, List<String>> parseQuery(String query) {
+    public static Map<String, List<String>> parseQuery(String query) {
```

Comment:
Should this be  moved into a utils class?

Also can  we document that it won't handle escape chars?